kyaml/runfn/runfn.go

```diff
@@ -92,6 +92,14 @@ type RunFns struct {
 
 	// Env contains environment variables that will be exported to container
 	Env []string
+
+	// ContinueOnEmptyResult configures what happens when the underlying pipeline
+	// returns an empty result.
+	// If it is false (default), subsequent functions will be skipped and the
+	// result will be returned immediately.
+	// If it is true, the empty result will be provided as input to the next
+	// function in the list.
+	ContinueOnEmptyResult bool
```

Comment:
This part went to a separate PR: https://github.com/kubernetes-sigs/kustomize/pull/3185
That PR also contains a UT for both cases